internal/controller/linodemachine_controller_helpers.go

```diff
@@ -690,7 +687,10 @@ func createDisks(ctx context.Context, logger logr.Logger, machineScope *scope.Ma
 	if err != nil {
 		return err
 	}
-	conditions.MarkTrue(machineScope.LinodeMachine, ConditionPreflightAdditionalDisksCreated)
+	conditions.Set(machineScope.LinodeMachine, metav1.Condition{
+		Type:   ConditionPreflightAdditionalDisksCreated,
+		Status: metav1.ConditionTrue,
```

Comment:
Do we need to set the Reason here like the others to not fail object patching?

Reply:
ah good catch let me add reason field here